RecyclerViewDiffUtilDataBinding-Starter/app/build.gradle

```diff
@@ -62,7 +62,7 @@ dependencies {
     implementation "androidx.room:room-runtime:$room_version"
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     kapt "androidx.room:room-compiler:$room_version"
-    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
+    implementation "androidx.lifecycle:lifecycle-extensions:2.0.0"
```

Comment:
how come it's down to 2.0.0?

Reply:
This need some code changes and may be codelab changes as well. So it is best to do it in a separate CL